src/utils/package-json.ts

```diff
@@ -101,9 +115,29 @@ function mergePackageJson(originalPackageJson: any, changes: any) {
   return adjustedPackageJson;
 }
 
-export async function modifyPackageJson(projectDir: string, isScaffold: boolean, frontendScripts: boolean, force: boolean) {
+export interface ModifyPackageJsonParams {
+  projectDir: string;
+  isScaffold: boolean;
+  frontendScripts?: boolean;
+  force?: boolean;
+  addCds?: boolean;
+}
+
+export async function modifyPackageJson({
+  projectDir,
+  isScaffold = false,
+  frontendScripts = false,
+  force = false,
+  addCds = false
+}: {
+  projectDir: string;
+  isScaffold?: boolean;
```

Comment:
This interface is almost `ModifyPackageJsonParams` why not use that?

Reply:
I wanted to delete it, just forgot to do it. ;)